templates/show-task.html

```diff
@@ -36,13 +39,17 @@
 
 {% block content %}
 	<div class="container">
+        <div id="alert-message-holder"></div>
 		<div class="row">
 			<div class="col-md-12 create-worklist-container">
-				<h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <h2 class="top-heading">Worklist Name: {{worklist_name}}</h2>
+                <a href="#" data-toggle="popover" data-content="Share Worklist using this link: https://tools.wmflabs.org/worklist-tool/show-tasks?worklist_name={{ worklist_name }}&worklist_created_by={{ worklist_created_by }}"><i class="fa fa-share-alt" class="share-icon-size"></i></a>
```

Comment:
I think we were discussing a shorter URL like on GitHub repositories, like:

https://tools.wmflabs.org/worklist-tool/worklist/{{ user }}/{{ worklist }}

How about that?

Reply:
Done.